zboxcore/zboxutil/http.go

```diff
@@ -162,7 +166,19 @@ func NewReferencePathRequest(baseUrl, allocation string, paths []string) (*http.
 	//url := fmt.Sprintf("%s%s%s?path=%s", baseUrl, LIST_ENDPOINT, allocation, path)
 	nurl.RawQuery = params.Encode() // Escape Query Parameters
 	req, err := http.NewRequest(http.MethodGet, nurl.String(), nil)
-	return setClientInfo(req, err)
+	if err != nil {
```

Comment:
I changed the `setClientInfo `method and removed the interaction with errors from it, due to the non-obvious behavior of the method.
I can undo the changes, but I think that the current method is more obvious. What do you think about this?

Reply:
yeah I get your point, unless it was wrapped in something else? not enough to block merges it's just a it of tech debt as changes will need made in 8 different places rather than 1